tensorflow/core/kernels/tensor_array.cc

```diff
@@ -51,6 +51,15 @@ TF_CALL_complex128(TENSOR_ARRAY_WRITE_OR_ADD_GPU);
 
 #endif  // GOOGLE_CUDA
 
+#ifdef TENSORFLOW_USE_SYCL
+
```

Comment:
For SYCL we didn't. Can we dedicate another PR to fixing the formating later on?